Makefile

```diff
@@ -0,0 +1,25 @@
+# Copyright 2015 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+.PHONY: all format clean master manager fuzzer executor
+
+all: master manager fuzzer executor
+
```

Comment:
I'm not sure if it make sense to copy config files from `make`.
A user will need to edit it anyway, and the instructions in README are explicit enough.